src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+
+table.tHead.addEventListener('click', (happening) => {
+  const item = happening.target;
+  const cell = item.cellIndex;
+  const row = [];
+
+  for (let i = 1; i < table.rows.length - 1; i++) {
+    const arreyOfvaluesOfCell = [];
```

Comment:
```suggestion
    const cellValuesArray = [];
```
Try to avoid using `of` twice